src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const mainImage = document.querySelector('#largeImg');
+const gallery = document.querySelector('.gallery');
+
+gallery.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  if (e.target.closest('.list-item__link')) {
```

Comment:
It's more efficient to store the result of `e.target.closest('.list-item__link')` in a variable and reuse it in line 10.